tests/testerina-integration-test/src/test/java/org/ballerinalang/testerina/test/CodegenCodeCoverageTest.java

```diff
@@ -105,16 +169,16 @@ private void validateCoverage() {
             for (JsonElement sourceFiles : moduleObj.get("sourceFiles").getAsJsonArray()) {
                 JsonObject fileObj = (JsonObject) sourceFiles;
                 if ("foo.bal".equals(fileObj.get("name").getAsString())) {
-                    Assert.assertEquals(parser.parse(Arrays.toString(fooCovered)),
-                            parser.parse(fileObj.get("coveredLines").getAsJsonArray().toString()));
-                    Assert.assertEquals(parser.parse(Arrays.toString(fooMissed)),
-                            parser.parse(fileObj.get("missedLines").getAsJsonArray().toString()));
+                    Assert.assertEquals(JsonParser.parseString(Arrays.toString(fooCovered)),
+                            JsonParser.parseString(fileObj.get("coveredLines").getAsJsonArray().toString()));
+                    Assert.assertEquals(JsonParser.parseString(Arrays.toString(fooMissed)),
+                            JsonParser.parseString(fileObj.get("missedLines").getAsJsonArray().toString()));
```

Comment:
Shall we use constants for `missedLines`, `coveredLines`?

Reply:
IMO its fine to keep `missedLines` and `coveredLines` as strings since moving only them to constants will make the code look bit inconsistent (because there are a set of other strings used in the same code block only once). If we make them all constants, then it will add a lot of bloat just for most of the constants to be used only once.